Content.Shared/Item/SharedItemComponent.cs

```diff
@@ -37,6 +37,20 @@ public int Size
         [DataField("clothingVisuals")]
         public Dictionary<string, List<PrototypeLayerData>> ClothingVisuals = new();
 
+        /// <summary>
+        ///     Whether or not this item can be picked up.
+        /// </summary>
+        /// <remarks>
+        ///     This should almost always be true for items. But in some special cases, an item can be equipped but not
+        ///     picked up. E.g., hardsuit helmets are attached to the suit, so we want to disable things like the pickup
+        ///     verb.
+        /// </remarks>
+        [DataField("canPickup")]
+        public bool CanPickup = true;
+
+        [DataField("quickEquip")]
+        public bool QuickEquip = true;
```

Comment:
shouldnt this be on clothing

Reply:
well i guess item and clothing are already insaenly fucked up and coupled so if it has to be like this thats okay

Reply:
It does also function for quickly moving items into pockets, so IMO its probably fine.